migrations/1691518766_add-suspension.sql

```diff
@@ -0,0 +1,5 @@
+alter table public.tenants
+    add suspended bool not null default false;
+
+alter table public.tenants
+    add suspended_reason text;
```

Comment:
Looks like we don't use this anywhere, so assuming this is for debugging/audit purposes only for now.

Reply:
We read it when we query the tenant and send it back to the cloud app to be displayed